commit 7dff6709506396873ed2240716459a0caf3dccac (refs/changes/23/623/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-10-07T10:35:29-07:00 (3 years ago)
    
    MANTA-2998 marlin's dev/package.json is broken by latest node-sdc-clients

diff --git a/common/lib/ufds.js b/common/lib/ufds.js
index d4ad1d2..7dfac94 100644
--- a/common/lib/ufds.js
+++ b/common/lib/ufds.js
@@ -41,7 +41,7 @@ exports.ufdsMakeAccounts = ufdsMakeAccounts;
  *     			if we develop a better approach to waiting on
  *     			replication), this dependency could be removed.
  *
- *     ufds		UFDS client object (from node-sdc-clients.git)
+ *     ufds		UFDS client object (from node-ufds.git)
  *
  *     config		A configuration object, which has this general form:
  *
diff --git a/dev/package.json b/dev/package.json
index dd50c2f..5266b14 100644
--- a/dev/package.json
+++ b/dev/package.json
@@ -40,7 +40,7 @@
 		"restify": "2.3.5",
 		"retry": "0.6.0",
 		"semver": "1.1.4",
-		"sdc-clients": "git+https://github.com/joyent/node-sdc-clients.git#master",
+		"ufds": "1.2.0",
 		"spawn-async": "0.0.1",
 		"statvfs": "2.0.0",
 		"strsplit": "1.0.0",
diff --git a/dev/test/common.js b/dev/test/common.js
index cdf6e77..afab3df 100644
--- a/dev/test/common.js
+++ b/dev/test/common.js
@@ -30,7 +30,7 @@ var mod_verror = require('verror');
 var mod_mahi = require('mahi');
 var mod_manta = require('manta');
 var mod_marlin = require('../lib/marlin');
-var mod_sdc = require('sdc-clients');
+var mod_ufds = require('ufds');
 var mod_maufds = require('../lib/ufds');
 
 var VError = mod_verror.VError;
@@ -97,7 +97,7 @@ function setup(callback)
 
 	barrier.start('ufds');
 	log.info({ 'url': ufds_url }, 'connecting to ufds');
-	ufds_client = new mod_sdc.UFDS({
+	ufds_client = new mod_ufds.UFDS({
 	    'url': ufds_url,
 	    'bindDN': 'cn=root',
 	    'bindPassword': 'secret'
